exercises/ngcourse-exercises/karma.conf.js

```diff
@@ -0,0 +1,48 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/0.13/config/configuration-file.html
+const getAppFromConfig = require('@angular/cli/utilities/app-utils').getAppFromConfig;
+module.exports = function (config) {
+  
+  appConfig = getAppFromConfig(config.angularCli.app);
```

Comment:
This change from the default CLI karma.conf lets us isolate tests to one sub-app